nrf_802154/zephyr/Kconfig.nrfxlib

```diff
@@ -32,7 +32,7 @@ config NRF_802154_SL
 	select DYNAMIC_INTERRUPTS
 	select SENSOR if NRF_802154_TEMPERATURE_UPDATE
 	select TEMP_NRF5_MPSL if NRF_802154_TEMPERATURE_UPDATE
-	select NRFX_PPI if SOC_SERIES_NRF52X
+	select NRFX_PPI if SOC_COMPATIBLE_NRF52X
```

Comment:
This file is deployed from
https://github.com/nrfconnect/sdk-nrf-802154/blob/main/zephyr/Kconfig.nrfxlib#L35
could you please make the change there?

Reply:
Thanks @ankuns Moved to  https://github.com/nrfconnect/sdk-nrf-802154/pull/375